package.json

```diff
@@ -43,9 +43,9 @@
   },
   "engines": {
     "node": ">=14.17.6",
-    "pnpm": ">=7.12.2 <=7.29.1"
+    "pnpm": ">=7.12.2 <=8.2.0"
   },
-  "packageManager": "pnpm@7.29.1",
+  "packageManager": "pnpm@8.2.0",
```

Comment:
We should create a separate pull request to upgrade the pnpm version,  please downgrade the pnpm version in this PR.

Reply:
thanks,I want to improve this place tomorrow